Header header
uint8 pin
uint8 value
